pkg/enqueue/Symfony/Consumption/ConsumeCommand.php

```diff
@@ -75,9 +76,12 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
             array_unshift($extensions, $loggerExtension);
         }
 
+        $captureExitStatusExtension = new CaptureExitStatusExtension();
+        array_unshift($extensions, $captureExitStatusExtension);
+
         $consumer->consume(new ChainExtension($extensions));
 
-        return null;
+        return $captureExitStatusExtension->getExitStatus();
```

Comment:
there is another consume command that has to be adjusted too
you should add a test for command too.